charts/consul/test/terraform/aks/variables.tf

```diff
@@ -7,7 +7,7 @@ variable "location" {
 }
 
 variable "kubernetes_version" {
-  default     = "1.27"
```

Comment:
suuuper minor but would be cool in the future if there was an AKS api we could hit or something and grab the version of this so we don't have to do it manually but that's way out of scope for this